src/epub/text/lazarus-come-forth.xhtml

```diff
@@ -53,7 +53,7 @@
 			<p>“Maybe⁠—just maybe⁠—we <em>have</em> got the weapon,” he said.</p>
 			<p>His hand trembled.</p>
 			<hr/>
-			<p>The coroner pumps throbbed warmly under the table, while manipulating tendrils darted swiftly, effectively over the dead Scientist’s body. Brandon moved, too, like a machine. In a regular fury he had forced Logan to hurry the body down into the preparations room, inject adrenalin, thermal units, apply the blood pump and accomplish a thousand other demanding and instantaneous tasks.</p>
+			<p>The coroner pumps throbbed warmly under the table, while manipulating tendrils darted swiftly, effectively over the dead Scientist’s body. Brandon moved, too, like a machine. In a regular fury he had forced Logan to hurry the body down into the preparations’ room, inject adrenalin, thermal units, apply the blood pump and accomplish a thousand other demanding and instantaneous tasks.</p>
```

Comment:
I don't think this is correct, this change implies the the room belongs to "the preparations", whatever that is? Isn't `preparations room` the name of the room in which preparations occur?

Unless this is some unusual plot point, I'd suggest that it's the other instances of `preparations’` that are incorrect, not this one.

Reply:
I agree.